sql/v2/types.go

```diff
@@ -34,6 +34,21 @@ func (t Type) String() string {
 	return "Any"
 }
 
+func (t Type) ZeroValue() interface{} {
+	switch t {
+	case StringType:
+		return ""
+	case IntegerType:
+		return 0
+	case BooleanType:
+		return false
+	case AnyType:
+		// by default, return false
+		return false
```

Comment:
I don't have a strong opinion on this one, but I'm curious why it defaults to "false" (here and before) instead of "nil" ?

Reply:
Or is it just because it needs to be of a well-known type, and "nil" doesn't fit that requirement?

Reply:
Throughout the spec whenever the return type is ambiguous we always have it defaulting to `Boolean`, which has a zero value of `false`. So, this way that works without much effort on our part in the SDK

Reply:
makes sense - thanks